lib/blocs/authentication/viewmodels/authentication_bloc.dart

```diff
@@ -35,12 +35,9 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
       add(const InitAuthStatus());
     }
     if (event is OnRecoverAccount) {
-      // TODO(RaulUrtecho): - This is redundant - user was already saved when recovery was started.
-      // also a recovery is 1:1 tied to a user account so the account can't change
-      // the key we use as argument here comes from settingsStorage and goes back there.
-      SaveAccountUseCase().run(accountName: event.account, privateKey: event.privateKey);
+      settingsStorage.finishRecoveryProcess();
       settingsStorage.privateKeyBackedUp = false;
-      // New account --> re-start auth status
+      // Recovery mode false --> re-start auth status
       add(const InitAuthStatus());
     }
```

Comment:
Should we set or remove the fcm token when the Recover Process is completed too?